book/01-guided-tour.html

```diff
@@ -29,24 +29,39 @@ <h1>A Guided Tour</h1>
     toplevel should mostly work fine.</p>
 
     <div data-type="note">
-      <h1><code>Core</code>, <code>Core_kernel</code> and
-      portability</h1>
-
-      <p><code>Core_kernel</code> is actually part of a more
-      expansive library called <code>Core</code>. We use
-      <code>Core_kernel</code> here because it is portable to every
-      platform that OCaml runs on, unlike <code>Core</code>, which
-      only works on Unix-like platforms, including Linux and MacOS.
-      If you want your code to run on Windows or Javascript, you
-      need to stick to <code>Core_kernel</code>.</p>
-
-      <p>You should check out <code>Core</code> if you'll be running
-      only on Unix. <code>Core</code> includes a lot of useful
-      functionality, including an excellent command line parsing
-      library and modules that provide access to low-level operating
-      systems facilities from mutexes to sockets. We'll discuss some
-      of the additional functionality provided by <code>Core</code>
-      later in the book.</p>
+      <h1><code>Base</code>, <code>Core</code> and <code>Core_kernel</code></h1>
+
+      <p><code>Base</code> is one of a family of three standard library
+      replacements, each with different use-cases. Here's a quick
```

Comment:
Note that they build one each other and aren't entirely standalone?  "Family" does imply that I guess.

Reply:
Adjusted to clarify.